core/src/main/resources/reference.conf

```diff
@@ -104,7 +104,7 @@ akka-persistence-jdbc {
 }
 
 # the akka-persistence-journal in use
-pg-journal {
+postgres-journal {
   class = "akka.persistence.postgres.journal.JdbcAsyncWriteJournal"
```

Comment:
I am not sure, but shouldnt we change that value to PostgresAsyncWriteJournal

Reply:
Agree, done!